time.go

```diff
@@ -35,7 +35,13 @@ func (t *Time) UnmarshalGraphQL(input interface{}) error {
 		t.Time = time.Unix(int64(input), 0)
 		return nil
 	case int64:
-		t.Time = time.Unix(input, 0)
+		if input >= 1e10 {
```

Comment:
I wonder if you need the the `if` statement at all... If `input` is smaller than `1e10` then `nsec` will be `0` anyway, no?